src/modules/robot/Robot.cpp

```diff
@@ -396,22 +396,20 @@ void Robot::on_gcode_received(void *argument)
                     this->max_speeds[Y_AXIS] = gcode->get_value('Y');
                 if (gcode->has_letter('Z'))
                     this->max_speeds[Z_AXIS] = gcode->get_value('Z');
-                if (gcode->has_letter('A'))
-                    alpha_stepper_motor->set_max_rate(gcode->get_value('A'));
-                if (gcode->has_letter('B'))
-                    beta_stepper_motor->set_max_rate(gcode->get_value('B'));
-                if (gcode->has_letter('C'))
-                    gamma_stepper_motor->set_max_rate(gcode->get_value('C'));
-
+                for (size_t i = 0; i < actuators.size(); i++) {
+                    if (gcode->has_letter('A' + i))
```

Comment:
That's good for readback, what should we do for entry? Specify an input/output : index and value? Like M203 I0 25 I2 15 O4 97? (OK, poor example. :) ) If this is a larger question, I'm OK backing out the g-code mods from this pull request and doing them separately as a follow-on once there's an agreed-on format. Also, are there guidelines somewhere for pinking new M-codes?